samples/crypto/aes_gcm/boards/nrf54l15pdk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=y
+
+# Note that we are turning off the AEAD driver and not the CRACEN
+# driver as a whole because we wish to specifically test AEAD with
+# Oberon, but still need the CRACEN driver for PRNG/TRNG.
```

Comment:
I guess there is a reason for this but samples should be tests, maybe we should make a Jira to follow this up